src/flash.h

```diff
@@ -29,6 +30,16 @@ typedef struct {
     rx_bayang_bind_data_t bayang;
     rx_unified_bind_data_t unified;
     rx_elrs_bind_data_t elrs;
+
+// TODO: For some reason adding this to the union actually changes the union size!
```

Comment:
that is.....perplexing.....brb, digging in the c standard

Reply:
https://github.com/BossHobby/QUICKSILVER/commit/5b83a1b86dcc613425232b6a2bedaab53824a9f0 should fix this issue

Reply:
Thank you. I updated my `develop` branch to obtain your fix, then rebased this feature branch on top of that. Then I
removed workaround code for bind storage data type weirdness. The `rx_flysky_bind_data_t` type member of the anonymous union in `rx_bind_storage_t` is now enabled.

See eadd532